torch/_export/converter.py

```diff
@@ -56,11 +56,19 @@ def get_op_overload(node: torch._C.Node):
     ns, op_name = str(schema.name.name).split("::")
     override = schema.name.overload_name
 
-    op_overload_packet = getattr(torch.ops.aten, op_name)
-    if override:
-        op_overload = getattr(op_overload_packet, override)
-    else:
-        op_overload = op_overload_packet.default
+    try:
+        op_overload_mod = getattr(torch.ops, ns)
+        op_overload_packet = getattr(op_overload_mod, op_name)
+        if override:
+            op_overload = getattr(op_overload_packet, override)
+        else:
+            op_overload = op_overload_packet.default
+    except Exception as e:
+        msg = f"Finding operator overload for {node.kind()}"
+        if override:
+            msg += f" with override {override}"
+        msg += f" fail due to: {e}"
```

Comment:
nit: `raise RuntimeError(f"Unable to find operator {node.kind()} with schema {node.schema}")`